iphone/templates/module/swift/template/ios/Classes/{{ModuleIdAsIdentifier}}Module.swift.ejs

```diff
@@ -40,11 +40,26 @@ class <%- moduleIdAsIdentifier %>Module: TiModule {
     super.startup()
     debugPrint("[DEBUG] \(self) loaded")
   }
+
+  @objc(example:)
+  func example(arguments: Array<Any>?) -> String {
+    // Example method. 
+    // Call with "MyModule.example(args)"
+
+    return "hello world!"
+  }
   
-  @objc(tryThis:)
-  func tryThis(arguments: Array<Any>?) -> String {
-    guard let arguments = arguments, let message = arguments.first else { return "No arguments" }
+  @objc(exampleProp)
```

Comment:
This would be a great place to demonstrate Swift property getter/setter instead of the ObjC style methods:

```swift
@objc public var exampleProp: String {
    get { 
        // Example property getter
        return "Titanium rocks!"
    }
    set {
        // Example property setter
        // Call with "MyModule.exampleProp = 'newValue'"
        self.replaceValue(newValue, forKey: "exampleProp", notification: false)
    }
}
```

Reply:
Good idea. Updated. Thanks!